tests/test_connection.py

```diff
@@ -153,6 +154,60 @@ def test_log_connection_metric_vectorization(self):
 
         self.assertAllClose(result, expected, rtol=1e-5, atol=1e-5)
```

Comment:
Sourcegraph spots these lines as not tested. Maybe deleting the geomstats.tests.main() at the end of this file will change this?

Reply:
I think the codecov report was generated before travis was done with the numpy tests, and that's why it says coverage will drop.